### PR TITLE
Update German translation

### PR DESCRIPTION
**Pull request by TheRealWormbo**
Major update to the translation file that covers almost all entries, including the entire lexicon.

### CODE REVIEW

**Comment by Kartabasss** [comment]
weeeeee

**Comment by TheRealWormbo** [comment]
Had to rebase to add another commit (14838b5), as the PR was from before the build script change and would likely not have compiled otherwise.

**Comment by TheRealWormbo** [comment]
How's the review coming along @Kartabasss?

File: Xplat/src/main/resources/assets/botania/lang/de_de.json [hunk 1]
```diff
@@ -5,52 +5,630 @@
...
+    "emi.category.botania.orechid_ignem": "Flammende Erchidee",
+    "emi.category.botania.marimorphosis": "Tagemorphes",
+
+    "tag.botania.petals.white": "Weiße Blütenblätter",
+    "tag.botania.petals.orange": "Orange Blütenblätter",
+    "tag.botania.petals.magenta": "Magentafarbene Blütenblätter",
+    "tag.botania.petals.light_blue": "Hellblaue Blütenblätter",
+    "tag.botania.petals.yellow": "Gelbe Blütenblätter",
+    "tag.botania.petals.lime": "Hellgrüne Blütenblätter",
+    "tag.botania.petals.pink": "Pinke Blütenblätter",
```
Comment (PssbleTrngle) [inline Xplat/src/main/resources/assets/botania/lang/de_de.json]: minor, but "pink" things usually get translated into "rosa" in minecraft, as seen in the vanilla translations as well (don't ask me why)

Comment (TheRealWormbo) [reply Xplat/src/main/resources/assets/botania/lang/de_de.json]: Good point, thank you.

**Pull request merged**